index.md

```diff
@@ -39,35 +29,24 @@ We recommend the following courses and skills:
 
 Nevertheless, this is a large, experimental course, so we do not expect specific background courses or experience. If you are unsure, use the textbooks and Unity manual as a reference for the level of rigor we expect, or ask course staff.
 
-## Lectures ##
-Lecture is held in 1320 Digital Computer Lab on MW 4-5:15pm
-
-## Assignments ##
-We expect to have 4 assignments over the course of the semester. In each of these, students will work in pairs to solve a specific implementation problem in the virtual reality lab in 4107 Siebel Center.
-
-See the [Assignments page](/assignments) for more details.
-
-## Final Project ##
-The final project is a semester-long implementation of a VR system. Students choose the topic and theme of the project under TA supervision.
-
-This satisfies the [team project requirement](https://cs.illinois.edu/academics/degree-program-options/bs-computer-science-engineering) for CS majors.
+## Course Staff ##
+See the [schedule for office hours](/officehours).
 
-See the [project gallery](/projects) for past student projects.
+* Professor Eric Shaffer, Siebel 2209
+* Professor Dan Cermak
 
-## Exams ##
-This class will have three midterm exams and no final exam. The third midterm will be just before finals. Each exam is 50 minutes long and will be held in the CBTF. You will need to make a reservation in the [CBTF](https://cbtf.engr.illinois.edu) to take the exam.
+## Lectures ##
+Lecture is held in 1320 Digital Computer Lab on MW 4-5:15pm
 
-- Midterm 1: Feb. 10 - Feb. 12
-- Optional second chance exam for Midterm 1: Feb. 17 - Feb. 19
-- Midterm 2: Mar. 10 - Mar. 12
-- Optional second chance exam for Midterm 2: Mar. 25 Mar. 26
-- Midterm 3:  Apr. 28 - Apr. 30
+## Textbooks ##
+Main text, required: [Steven M. LaValle, Virtual Reality, 2016](http://vr.cs.uiuc.edu).
 
-All exams are closed-everything (no book, notes, calculator, etc.). See the [CBTF policies](https://cbtf.engr.illinois.edu/for-students/policies.html) for more detail.You are responsible for all material covered in lectures and assignments.For DRES accommodations see [CBTF DRES Students](https://cbtf.engr.illinois.edu/for-students/dres-students.html)
+See the [Extra Material](/extra-material) page for other, recommended readings.
 
-## Second-chance Exams ##
+## [Piazza](https://piazza.com/illinois/spring2019/cs498vr) ##
```

Comment:
Maybe we should move the Piazza link, and everything else that is semester-dependent to variables in `_config.yml`? That way when we're updating the site for a new semester, we can see everything we need to update in one place.

Reply:
Added to _config.yml